stdlib/experimental/json/json_test.flux

```diff
@@ -8,38 +8,35 @@ option now = () =>
 	(2030-01-01T00:00:00Z)
 
 inData = "
-#datatype,string,long,dateTime:RFC3339,string,string
-#group,false,false,false,false,false
-#default,_result,,,,
-,result,table,_time,_field,_value
-,,0,2018-05-22T19:53:26Z,json,\"{\"\"a\"\":1,\"\"b\"\":2,\"\"c\"\":3}\"
-,,0,2018-05-22T19:53:36Z,json,\"{\"\"a\"\":2,\"\"b\"\":4,\"\"c\"\":6}\"
-,,0,2018-05-22T19:53:46Z,json,\"{\"\"a\"\":3,\"\"b\"\":5,\"\"c\"\":7}\"
+#datatype,string,long,dateTime:RFC3339,string,string,string
+#group,false,false,false,true,true,false
+#default,_result,,,,,
+,result,table,_time,_field,_measurement,_value
```

Comment:
The input data needs to have `_measurement` so it can be properly written to storage when running in OSS.